src/app/routes/app-routes.tsx

```diff
@@ -260,6 +264,20 @@ function AppRoutesAfterUserHasConsented() {
             </AccountGate>
           }
         />
+        <Route
+          path={RouteUrls.RpcSendTransfer}
+          element={
+            <AccountGate>
+              <RpcSendTransfer />
+            </AccountGate>
+          }
+        />
+        <Route path={RouteUrls.RpcSendTransferChooseFee} element={<RpcSendTransferChooseFee />} />
+        <Route
+          path={RouteUrls.RpcSendTransferConfirmation}
+          element={<RpcSendTransferConfirmation />}
+        />
+        <Route path={RouteUrls.RpcSendTransferSummary} element={<RpcSendTransferSummary />} />
```

Comment:
Maybe it'd make sense to have all these under a container route?

Reply:
Yep, def will revisit and do today.